instrumentation/apache-dubbo/apache-dubbo-2.7/javaagent/src/main/resources/META-INF/services/org.apache.dubbo.rpc.Filter

```diff
@@ -0,0 +1 @@
+io.opentelemetry.javaagent.shaded.instrumentation.apache.dubbo.v2_7.OpenTelemetryFilter
```

Comment:
In should be lived in library in order to make it can be autoload when user only use the library.

Reply:
oh, yes, I missed that it's in both places. you should be able to delete the one in the javaagent instrumentation, and it will use (and shade appropriately) the one from the library instrumentation

Reply:
oh, it works! This is really beyond my expectation. I think I should  read the source code of this set carefully during the Chinese Spring Festival holiday.

Reply:
It seemed that Task :instrumentation:aws-lambda-1.0:library:compileJava failed.

Reply:
Oh no looks like main branch is busted

Reply:
sorry about that! main is fixed now by #2213

Reply:
ah, it is ok now :)